.github/workflows/python-tests.yml

```diff
@@ -35,8 +35,30 @@ jobs:
       - name: Create virtual environment and install dependencies
         run: |
           uv venv
-          uv sync
+          uv sync --dev  # Include dev tools like pytest and pytest-cov
 
-      - name: Run tests
+      - name: Run tests and generate Cobertura coverage report
         run: |
-          uv run pytest
+          mkdir -p coverage
+          PYTHONPATH=. uv run pytest \
+            --cov=auth \
```

Comment:
Do we have to specify directories to do coverage on manually? Might be annoying to update in future